frontend/app/src/components/MultiToggle.svelte

```diff
@@ -0,0 +1,63 @@
+<script context="module" lang="ts">
+    export type Option = {
+        id: string;
+        label: string;
+    };
+</script>
+
+<script lang="ts">
+    import { _ } from "svelte-i18n";
+
+    export let options: Option[];
+    export let selected: string;
+
+    function selectOption(id: string) {
+        selected = id;
+    }
+</script>
+
+<div class="toggle-wrapper" role="radiogroup">
+    {#each options as { id, label }, i}
+        <div
+            role="radio"
+            tabindex={i}
+            class="toggle"
+            class:selected={id === selected}
+            class:first={i === 0}
+            class:last={i === options.length - 1}
+            on:click={() => selectOption(id)}>
+            {label}
+        </div>
+    {/each}
+</div>
+
+<style lang="scss">
+    .toggle-wrapper {
+        display: flex;
+        align-items: center;
+        border: 1px solid var(--bd);
+        border-radius: 4px;
+        cursor: pointer;
+
+        .toggle {
+            border: none;
+            border-right: 1px solid var(--bd);
+            padding: 6px 10px 5px 9px;
+
+            &:hover,
```

Comment:
button has a different background colour variable for hover so maybe we should use the same here?